.eslintrc.json

```diff
@@ -9,27 +9,19 @@
 
   "env": {
     "browser": true,
-    "node": true,
-    "mocha": true
+    "node": true
   },
 
   "globals": {
-    "Map": true,
-    "Set": true,
-    "Symbol": true,
-    "chrome": true,
-    "Promise": true,
-    "Uint8Array": true,
-    "ArrayBuffer": true,
-    "FileReaderSync": true,
-    "sqlitePlugin": true,
-    "emit": true,
-    "PouchDB": true,
-    "should": true,
-    "assert": true,
-    "testUtils": true,
-    "importScripts": true,
-    "testCases": true
+    "Map": "readonly",
```

Comment:
Shouldn't we just bump the ecmaVersion rather than allowing a selection of globals?

Reply:
That would probably be a useful change too, but I think we'd still need quite a few of the globals.  This change isn't that drastic - it's making the existing eslint globals config more restrictive, and updating to modern syntax.